### ISSUE TITLE
Adding TensorFlow port of LeViT

### ISSUE THREAD

**Issue by ariG23498** [huggingface/transformers#19123]
### Feature request

To add the TensorFlow port of the [LeViT](https://arxiv.org/abs/2104.01136) architecture. The architecture is currently present in the Transformers library in [PyTorch](https://github.com/huggingface/transformers/blob/main/src/transformers/models/levit/modeling_levit.py). 

### Motivation

[LeViT](https://arxiv.org/abs/2104.01136) is a family of architectures that optimize the trade-off between accuracy and efficiency in a high-speed regime. The TensorFlow port would be an addition to the hybrid architecutre families.

### Your contribution

I would like to make the contribution by building out the TensorFlow port.

Tagging: @amyeroberts who could assign me to the task of adding the TensorFlow port of the model.

**Comment by ariG23498** [comment]
I am still working on this.
